gatk.wdl

```diff
@@ -194,3 +194,26 @@ task CombineGVCFs {
         File output_gvcf_index = output_basename + ".vcf.gz.tbi"
     }
 }
+
+task SplitNCigarReads {
+    String? preCommand
+
+    File input_bam
+    File ref_fasta
```

Comment:
GATK does require a dict file next to the fasta file, this should be included as input file